### DISCUSSION TITLE
Allow installation of Chrome versions

### DISCUSSION THREAD

**Discussion by platformbeheer-otv** [chocolatey-community/chocolatey-packages#2361] [General]
Currently the Chrome package only installs the latest version. In June Google added support for downloading specific versions, see:
https://developer.chrome.com/blog/chrome-for-testing/

This could be interesting for the Chrome Chocolatey package to allow --version support.

**Comment by pauby** [comment]
If you're willing to put the work in to update the package scripts, I think it's a great idea.

**Comment by TheCakeIsNaOH** [comment]
From that linked page: 
> We intentionally do not list Chrome for Testing on [the Google Chrome download page](https://www.google.com/chrome/), because it is not meant to be used for regular browsing by regular users.

I think a new package should be created, but I don't think the normal Google Chrome package should be updated to use these binaries, as they are different from the Chrome that Google intends for most users. For example, they are shipped as zip archives, not installers.

**Reply by pauby** [reply]
I didn't look at that link. Thanks for picking up on this @TheCakeIsNaOH. Agreed, this should **not** replace the current package.

If a new cc package is being created then it makes sense for it to live here but it doesn't have to. If it does live here, whoever creates it needs to agree to support it (through CODEOWNERS file changes).
